<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>798</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>786</width>
    <height>798</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>786</width>
    <height>798</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Steganosaure</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#main_widget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(144, 207, 255, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton {
  background-color: white; 
  color: black; 
  border: 2px solid #008CBA;
  font-size:13px;
  font-weight:bold;
  border-radius: 10px;
}
QPushButton:hover {
  background-color: #008CBA;
  color: white;
}

QPlainTextEdit{
  border-radius: 10px;
}

QLabel#title_label{
color: rgb(31, 69, 152);
}
</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>131</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Parcourir</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>381</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="message_plain_text_edit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>590</y>
      <width>384</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="watermark_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>730</y>
      <width>136</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>steganographier</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Steganosaure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extract_message_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>730</y>
      <width>149</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Extraire le message</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
